x-pack/plugin/sql/qa/server/src/main/resources/date_nanos.csv-spec

```diff
@@ -314,7 +314,7 @@ SELECT id, "@timestamp"::string AS ts_string FROM logs_nanos WHERE id < 20 ORDER
 
 orderByScalar
 schema::id:i|ts_string:s
-SELECT id, "@timestamp"::string AS ts_string FROM logs_nanos ORDER BY DATE_PART('nanoseconds', "@timestamp") DESC LIMIT 10;
+SELECT id, "@timestamp"::string AS ts_string FROM logs_nanos ORDER BY DATE_PART('nanoseconds', "@timestamp") DESC NULLS LAST LIMIT 10;
```

Comment:
`NULL` now shows up first if you don't specify `NULLS LAST` with `DESC` direction.